server/server.go

```diff
@@ -500,7 +508,7 @@ func (s *server) buildRegistryInfo(lAddr net.Addr) {
 		info.ServiceName = s.opt.Svr.ServiceName
 	}
 	if info.PayloadCodec == "" {
-		info.PayloadCodec = s.svcs.defaultSvc.svcInfo.PayloadCodec.String()
+		info.PayloadCodec = getDefaultSvc(s.svcs).svcInfo.PayloadCodec.String()
```

Comment:
If multiple payload codecs are registered with the server, this payload codec may differ from the actual payload codec of the request.
Should I change the string type Info.PayloadCodec to []string?